Makefile

```diff
@@ -367,7 +367,7 @@ clang_setup:
 	    echo '$(TERM_INFO)(TIP: you can also force a clang rebuild by removing $(FCP_DIR)/clang/installed.version)$(TERM_RESET)' >&2; \
 	    echo >&2 ; \
 	  fi; \
-	  $(FCP_DIR)/clang/setup.sh; \
+	  $(FCP_DIR)/clang/setup.sh $(FCP_CLANG_USE_NINJA) $(FCP_CLANG_SEQUENTIAL_LINK); \
```

Comment:
Makefile variables that are used during the build like that are usually filled in by `./configure`. It would even allow us to auto-detect if `ninja` is available and then we won't need to add any options to `./build-infer.sh`.

Reply:
Done!